tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "es2019",
+    "module": "commonjs",
+    "outDir": "./lib",
+    "rootDir": "./src",
+    "strict": true,
+    "noImplicitAny": false,
```

Comment:
I understand this configuration was copied from existing projects. However, I suggest setting `noImplicitAny` to `true` in this project to take full advantage of TypeScript's type checking. Otherwise, we're not getting much benefit over plain JavaScript.